packages/@ourworldindata/grapher/src/stackedCharts/MarimekkoChart.tsx

```diff
@@ -579,16 +576,9 @@ export class MarimekkoChart
         fontSize: number,
         isSelected: boolean
     ): LabelCandidate {
-        let label = item.entityName
-        let labelBounds = Bounds.forText(label, {
-            fontSize,
-        })
-        if (labelBounds.width > SOFT_MAX_LABEL_WIDTH && item.shortEntityName) {
-            label = item.shortEntityName
-            labelBounds = Bounds.forText(label, {
-                fontSize,
-            })
-        }
```

Comment:
We used to measure the original label first, but I think it's ok to just use the shorter version in the chart area if available